app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -296,6 +296,13 @@ sealed class Event {
             get() = hashMapOf(Events.appOpenedKeys.source to source.name)
     }
 
+    data class AppRecievedIntent(val source: Source) : Event() {
```

Comment:
nit: can we correct the spelling, in follow-up to avoid re-running CI? Should be `Received`. Here and in several other places

Reply:
sorry I tried hard to teach myself its "e" before "i"... but starting to think there is a bigger issue here 😅